lambdas/favourite/src/favourite-handler.ts

```diff
@@ -42,7 +48,33 @@ export class FavouriteLambda implements LambdaInterface {
     if (sessionService.hasDateExpired(sessionItem.expiryDate)) {
       throw new SessionExpiredError();
     }
+
+    const auditEventContext = this.createAuditEventContext(
+      sessionItem,
+      toyBody.toy
+    );
+
+    await this.auditService.sendAuditEvent(
+      AuditEventType.REQUEST_SENT,
+      auditEventContext
+    );
     const response = await this.callExternalToy(toyBody.toy);
+
+    const toyResponse =
+      (response.status !== 200 ? "ToyNotFoundError: " : "Toy found: ") +
```

Comment:
What about other status codes? Maybe we should flip the condition

Reply:
Seems there's only 200 and 404 defined - https://github.com/alphagov/di-ipv-third-party-stubs/blob/e2104ea4cc01b7edb277fa73af0530bc024805d0/stubs/toy/config/openapi-spec.yaml#L19, but I guess you can always get a 500 or 403 so I'll add specificity for those 👍

Reply:
Consider `if(200) { toy found } else 404 { toynotfound } else { error }`